### ISSUE TITLE
[Feature Request] - Movement detection

### ISSUE THREAD

**Issue by willowbank48** [mendhak/gpslogger#1108]
If there is movement detected based on gps speed, then increase or decrease the occurrences of GPS loging.
For more flexible battery management.   
   
Eg. If the timer is set to check and log the gps location at 3 minute intervals, but upon detecting movement at more than 5km/h increase this to every 15 seconds, then if speed drops below 5km/h decrease to once every 30 seconds for 3 minutes then drop to once every 3 minutes.

**Comment by rekcodocker** [comment]
Similar to #1115 I think.

**Comment by codethief** [comment]
@rekcodocker No, while the tickets are somewhat related, their are about different issues. The one you linked is about adjusting the logging frequency (to an external file) and decoupling it from the location polling frequency. The present ticket is about adjusting the location polling frequency.

**Comment by codethief** [comment]
Back to the original topic, I'd like to add that one might also be able to use the accelerometer and other sensors to detect ongoing movement.
